scapy/layers/tls/automaton_srv.py

```diff
@@ -796,6 +923,45 @@ def WAITING_CLIENTDATA(self):
     def RECEIVED_CLIENTDATA(self):
         pass
 
+    def save_ticket(self, ticket):
+        """
+        This function save a ticket and others parameters in the
+        file given as argument to the automaton
+        Warning : The file is not protected and contains sensitive
+        information. It should be used only for testing purpose.
+        """
+        if (not isinstance(ticket, TLS13NewSessionTicket) or
+                self.session_ticket_file is None):
+            return
+
+        s = self.cur_session
+        with open(self.session_ticket_file, "ab") as f:
+            # ticket;ticket_nonce;obfuscated_age;start_time;resumption_secret
```

Comment:
Why not using a comma to delimit the fields?

Reply:
It's clearer ? It matters really little, this is only used internally

Reply:
I think that it is better to use a well-known format even for internal usages =)